blocks/topnews/_topnews.sass

```diff
@@ -0,0 +1,10 @@
+.topnews
+  display: block
+  @media all and (min-width: 1025px)
+    grid-template-columns: 1fr 1fr 1fr
+    display: grid
+
+.topnews__news
+.topnews__news__date
+.topnews__news__title
+.topnews__news__abstract
```

Comment:
Not blocking issue: this could be commented since a linter may complain about empty blocks